README.md

```diff
@@ -77,6 +77,14 @@ See <https://github.com/ansible/ansible/issues/71528> for more information.
 - The ZooKeeper service can be started via: `systemctl start zookeeper`
 - The ZooKeeper service can be stopped via: `systemctl stop zookeeper`
 
+## Which zookeeper is the leader ?
```

Comment:
Could rephrase and give more context to the command being run. For example:

## Four Letter Word Commands

ZooKeeper can use commands based on four letter words, see <https://zookeeper.apache.org/doc/current/zookeeperAdmin.html#sc_4lw>

The below example uses the `stat` command to find out which instance is the leader.

```bash
for i in 1 2 3 ; do
  echo "zookeeper0$i is a "$(echo stat | nc zookeeper0$i 2181 | grep ^Mode | awk '{print $2}'); 
done
```

Reply:
Done!

Reply:
How do we solve this markdown lint tyranny ? :|

Reply:
You referring to the linting rules around line length etc? Do you use an editor like VSCode? Markdown linting extensions should show violations, and formatting/auto-format on save actions will resolve.